docker/_fail-debian-buster-amd64-openssl-1.1.x/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@prisma/debian-buster-amd64-openssl-undefined",
```

Comment:
Long term Prisma should throw an installation when no OpenSSL is found I assume instead of failing later?